.circleci/config.yml

```diff
@@ -49,7 +49,7 @@ jobs:
       - run: gem install bundler -v 2.2.10
       - run: bundle _2.2.10_ install
       - run: mkdir ./rspec
-      - run: bundle _2.2.10_ exec rspec --format progress --format RspecJunitFormatter -o ./rspec/rspec.xml spec
+      - run: bundle _2.2.10_ exec rspec --format documentation --format RspecJunitFormatter -o ./rspec/rspec.xml spec
```

Comment:
This just makes the test output show the actual names of the tests rather than `....`.

Reply:
Definitely useful for telling which tests are failing and what's going on while debugging.